offload_vio/device_rx/Makefile

```diff
@@ -1,5 +1,3 @@
-nproc=$(shell python3 -c 'import multiprocessing; print( max(multiprocessing.cpu_count() - 1, 1))')
-
```

Comment:
Why the deletion?

Reply:
Oh I interpreted this comment as omit from this PR. Should I keep it in the Makefiles?
https://github.com/ILLIXR/ILLIXR/pull/331#discussion_r864462123

Reply:
Yup, keep it around! I had meant that in the future the _entire_ Makefile could be included from common.